app/src/components/common/CardSignInRequired.vue

```diff
@@ -118,6 +121,7 @@ h3 {
   padding: 0 1rem 5rem;
   margin: 0 auto;
   width: 100%;
+  overflow: auto;
```

Comment:
why is this needed?

Reply:
CardSignInRequired was "losing" the top. It wasn't being displayed

Reply:
the session modal should handle this though